cmd/gitbase/main.go

```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"os"
 
-	"github.com/src-d/gitbase/cli/gitbase/cmd"
+	"github.com/src-d/gitbase/cmd/gitbase/cmd"
```

Comment:
maybe we should move the contents of the cmd folder into the gitbase folder. It does not make much sense to have them in a folder and the path looks weird